docs/current/guides/967042-customize-engine-configuration.md

```diff
@@ -0,0 +1,53 @@
+---
+slug: /967042/customize-engine-configuration
+displayed_sidebar: 'current'
+category: "guides"
+tags: ["docker"]
+authors: ["Tom Chauveau"]
+date: "2023-09-26"
+---
+
+# Customize Dagger Engine configuration
+
+## Introduction
+
+This guide explains how to configure Dagger Engine through the custom `engine.toml` file.
+
+For example, this can be used to configure a registry mirror for pulling container images instead of DockerHub.
+
```

Comment:
Can you please add a "requirements" section here? similar to https://docs.dagger.io/882813/build-test-publish-java-spring#requirements, updating as needed.

Reply:
Added